homeassistant/components/nzbget/sensor.py

```diff
@@ -32,7 +33,7 @@
         key="AverageDownloadRate",
         name="Average Speed",
         device_class=SensorDeviceClass.DATA_RATE,
-        native_unit_of_measurement=UnitOfDataRate.MEGABYTES_PER_SECOND,
+        native_unit_of_measurement=UnitOfDataRate.BYTES_PER_SECOND,
```

Comment:
This actually will change it for our existing users, unless we set a suggested unit to match the old one.

Reply:
To what should it be set? It was set to MEGABYTES_PER_SECOND, while the value was MEBIBYTES_PER_SECOND

Reply:
Yes, the native value provided by the API is `BYTES_PER_SECOND`, while the suggested unit for the user will be `MEGABYTES_PER_SECOND`. This makes `MEGABYTES_PER_SECOND` the default (while the integration just provides `BYTES_PER_SECOND`)